src/main/java/net/mcreator/ui/minecraft/DataListComboBox.java

```diff
@@ -108,7 +109,7 @@ public Component getListCellRendererComponent(JList<? extends DataListEntry> lis
 				Icon imageIcon = getIcon();
 				if (imageIcon instanceof ImageIcon)
 					setIcon(ImageUtils.changeSaturation((ImageIcon) imageIcon, 0.1f));
-				setText("<html>" + getText() + "<small> - not supported");
+				setText(L10N.t("dialog.list_field.data_not_supported", getText()));

```

Comment:
This one is not from list field, nor dialog, so change translaiton key to something like datalist_combobox.not_supported

Reply:
Done.